### PR TITLE
[OSP] Deprecate computeFlavor in install-config.yaml

### PR DESCRIPTION
**Pull request by maxwelldb**
Implements [OSDOCS-1716](https://issues.redhat.com/browse/OSDOCS-1716)

### CODE REVIEW

**Review by Fedosin — CHANGES_REQUESTED** [review]
The key is called `type`, not `flavorName`: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L288

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -412,6 +412,9 @@ ifdef::osp[]
 
 |`platform.openstack.computeFlavor`
 |The {rh-openstack} flavor to use for control plane and compute machines.
+
+This property is deprecated. To use a flavor as the default for all machine pools, add it as the value of the `flavorName` key in the `platform.openstack.defaultMachinePlatform` property. You can also set a flavor value for each machine pool individually.
```
Comment (maxwelldb) [inline modules/installation-configuration-parameters.adoc:416]:
```suggestion
This property is deprecated. To use a flavor as the default for all machine pools, add it as the value of the `type` key in the `platform.openstack.defaultMachinePlatform` property. You can also set a flavor value for each machine pool individually. 
```

**Comment by maxwelldb** [comment]
> The key is called `type`, not `flavorName`: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L288

TY. Changed in https://github.com/openshift/openshift-docs/pull/28797/commits/c05fe0ac2c7af8e7adfe55641c064036967280da

**Review by Fedosin — APPROVED** [review]
/lgtm

**Comment by maxwelldb** [comment]
Thanks! @Fedosin

**Comment by maxwelldb** [comment]
Should get a line in #28504

**Comment by maxwelldb** [comment]
Preview link: https://osp-deprecate-computeflavor-osdocs1716--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-configuration-parameters_installing-openstack-installer-custom

**Comment by wjiangjay** [comment]
@maxwelldb Sorry for the late, LGTM.

**Comment by maxwelldb** [comment]
@wjiangjay Thanks!

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL. Thanks!

**Comment by ahardin-rh** [comment]
LGTM! Just need to squash the commits and you're all set. :rocket:

**Pull request merged**